docs/manual/scripts.rst

```diff
@@ -0,0 +1,35 @@
+.. _scripts:
+
+Scripts
+=======
+
+Dump data
+---------
+
+Met het script `dump_data.sh` kan de data van alle componenten (zaken, documenten...) worden geëxporteerd naar een sql bestand.
```

Comment:
Otherwise it's rendered as *italic* in rst
```suggestion
Met het script ``dump_data.sh`` kan de data van alle componenten (zaken, documenten...) worden geëxporteerd naar een sql bestand.
```